pyworkflow/project/project.py

```diff
@@ -1957,37 +1957,31 @@ def setReadOnly(self, value):
         self.settings.setReadOnly(value)
 
     def fixLinks(self, searchDir):
-        logger.info("Fixing project links. Searching at %s" % searchDir)
+        logger.info(f"Fixing links for project {self.getShortName()}. Searching in: {searchDir}")
         runs = self.getRuns()
 
+        counter = 0
         for prot in runs:
-            print (prot)
-            broken = False
-            if isinstance(prot, ProtImportBase) or prot.getClassName() == "ProtImportMovies":
-                logger.info("Import detected")
-                for _, attr in prot.iterOutputAttributes():
-                    for f in attr.getFiles():
-                        if ':' in f:
-                            f = f.split(':')[0]
-
-                        if not os.path.exists(f):
-                            if not broken:
-                                broken = True
-                                logger.info("Found broken links in run: %s" %
-                                      pwutils.magenta(prot.getRunName()))
-                            logger.info("  Missing: %s" % pwutils.magenta(f))
-
-                            if os.path.islink(f):
-                                sourceFile = os.path.realpath(f)
-                                logger.info("    -> %s" % pwutils.red(sourceFile))
-
-                                newFile = pwutils.findFile(os.path.basename(sourceFile),
-                                                       searchDir,
-                                                       recursive=True)
-                                if newFile:
-                                    logger.info("  Found file %s, creating link... %s" % (newFile,
-                                        pwutils.green("   %s -> %s" % (f, newFile))))
-                                    pwutils.createAbsLink(newFile, f)
+            if prot.getClassName().startswith("ProtImport"):
```

Comment:
Not sure if all import protocols stars with ProtImport... but probably not all inherit from ProtImportBase

Reply:
ok, I will check all of them. I have not found any protocol inherited from ProtImportBase from pw. All are inherited from ProtImportImages from pwem, but we don't want to import pwem here..

Reply:
All import protocols in pwem and tomo start with ProtImport